src/envo/envo-edit.owl

```diff
@@ -40138,6 +40140,37 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105019> "quality of a foam"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105019> <http://purl.obolibrary.org/obo/PATO_0001018>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105022> (fouling)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Fouling>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.corrosionpedia.com/definition/540/fouling>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105022> "A material accumulation process during which materials accumulate on solid surfaces in a way that reduces the efficiency of the affected part."@en)
```

Comment:
>A material accumulation process during which materials accumulate on solid surfaces in a way that reduces the efficiency of the affected part.

A material accumulation process during which material builds up on a solid surface such that a function of that surface, or the object it is a part of, is impaired.

For a comment: Following BFO:function semantics, this suggests that the functions of the object of which the surface is a part are affected in their realisation in one or more processes.